src/custom_nodes/horizontal_ocr/README.md

```diff
@@ -0,0 +1,51 @@
+Additionally to the detected text boxes, in the two additional outputs are returned their coordinates and confidence levels.  
+
+# Building custom node library
+
+You can build the shared library of the custom node simply by running command in this custom node folder context:
+```
+make
+```
+It will compile the library inside a docker container and save the results in `lib` folder.
+
+You can also select base OS between RH 8.4 (redhat), CentOS 7 (centos) and Ubuntu 20.04 (ubuntu) by setting `BASE_OS` environment variable.
+```
+make BASE_OS=redhat
+```
+
+# Custom node inputs
+
+| Input name       | Description           | Shape  | Precision |
+| ------------- |:-------------:| -----:| ------:|
+| image      | Input image in an array format. Only batch size 1 is supported and images must have 3 channels. Resolution is configurable via parameters original_image_width and original_image_height. | `1,3,H,W` | FP32 |
+| boxes      | horizontal-text-detection model output `boxes` | `100,5` | FP32 |
+
+
+# Custom node outputs
+
+| Output name        | Description           | Shape  | Precision |
+| ------------- |:-------------:| -----:| -------:|
+| text_images      | Returns images representing detected text boxes. Boxes are filtered based on confidence_threshold param. Resolution is defined by the node parameters. All images are in a single batch. Batch size depend on the number of detected objects.  | `N,1,C,H,W` | FP32 |
+| text_coordinates      | For every detected box `N` the following info is added: x coordinate for the box center, y coordinate for the box center, box original width, box original height | `N,1,4` | I32 |
+| confidence_levels |   For every detected box `N` information about score result | `N,1,1` | FP32 |
+
+# Custom node parameters
+
+| Parameter        | Description           | Default  | Required |
+| ------------- | ------------- | ------------- | ----------- |
+| original_image_width  | Required input image width |  | &check; |
+| original_image_height  | Required input image height |  | &check; |
+| original_image_layout  | Input image layout. Possible layouts: NCHW/NHWC. When using NHWC, it is possible to accept binary inputs. | NCHW | |
+| target_image_width | Target width of the text boxes in output. Boxes in the original image will be resized to that value.  |  | &check; |
+| target_image_height  | Target width of the text boxes in output. Boxes in the original image will be resized to that value. |  | &check; |
+| target_image_layout  | Output images layout. Possible layouts: NCHW/NHWC. When using NHWC, it is possible to accept binary inputs. | NCHW | |
+| convert_to_gray_scale  | Defines if output images should be in grayscale or in color  | false | |
+| confidence_threshold | Number in a range of 0-1 |  | &check; |
+| debug  | Defines if debug messages should be displayed | false | |
+| max_output_batch  | Prevents too big batches with incorrect confidence level. It can avoid exceeding RAM resources | 100 | |
```

Comment:
Add a link to camera example to show how it can be tested

Reply:
Link added